test/visual/default.html

```diff
@@ -1,27 +1,40 @@
 <!DOCTYPE html>
-
-<head lang="en">
-  <meta charset="UTF-8">
-  <title></title>
-  <script src="../../bower_components/webcomponentsjs/webcomponents-lite.js"></script>
-  <script>
-    const theme = window.location.search.replace(/.*theme=(\w+).*/, '$1') || 'lumo';
-    document.write(`<link rel="import" href="../../theme/${theme}/vaadin-progress-bar.html">`);
-
-    window.addEventListener('WebComponentsReady', function() {
-      customElements.whenDefined('vaadin-progress-bar').then(function() {
-        window.ShadyDOM && window.ShadyDOM.flush(); // Force DOM composition
-        window.webComponentsAreReady = true; // Checked in gemini before capture callback
+<html>
+  <head lang="en">
+    <meta charset="UTF-8">
+    <title>Progress bar default tests</title>
+  
+    <script>
+      window.addEventListener('WebComponentsReady', function() {
+        customElements.whenDefined('vaadin-progress-bar').then(function() {
+          window.ShadyDOM && window.ShadyDOM.flush(); // Force DOM composition
+          window.webComponentsAreReady = true; // Checked in gemini before capture callback
+        });
       });
```

Comment:
Please remove this block, it does nothing. 
The `WebComponentsReady` event is dispatched by polyfill and now we removed it.